packages/upstream/tar-unix.0.9.0/opam

```diff
@@ -22,4 +22,3 @@ depends: [
   "lwt"
 ]
 available: [ ocaml-version >= "4.03.0" ]
-patches: [ "0001-Do-not-always-build-with-safe-string-this-can-break-.patch" ]
```

Comment:
Should you put the patch back in here as well?

Reply:
:O well spotted. I was tricked by the other re-added patch :P

Reply:
No, I take it back. I see it in the latest commit

Reply:
I have added it back - see the last commit. Well spotted.